etc/adminhtml/system.xml

```diff
@@ -184,6 +184,21 @@
                         <config_path>payment/omise_offline_paynow/title</config_path>
                     </field>
                 </group>
+                <group id="omise_offline_promptpay" translate="label" sortOrder="160" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
I'm not quite sure how `sortOrder` number works but it seems the number is incorrect since the PayNow node. (guess it should start after `310`)